taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+            </td>
+          </tr>
+          <tr>
+            <td>Graphical Tool</td>
+            <td>
+              <a target="_blank" href="https://cyberduck.io/">Cyberduck</a>
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <h4 id="storage">What is TACC's Storage Server?</h4>
+
+      <p>
+        A TACC Storage System is one of TACC’s systems dedicated to storage and data management. It can be used for storing protected data on a secure location for this Portal. As an Portal user, you have access to a secure server for this Portal, like “
+        <code><samp>sub.domain</samp>.tacc.utexas.edu</code>
+        ” (which this document will refer to as <kbd>host</kbd>), as well as the secure folder for the Portal, like “
+        <code>/<kbd>secure-server-root</kbd>/projects/<kbd>PORTAL</kbd></code>
+        ” (which this document will refer to as <kbd>/transfer/directory/path</kbd>). This secure system is to be used exclusively for transferring and accessing Portal data.
+      </p>
+      <h4 id="rsync_prereqs">Prerequisites for Portal User</h4>
+
+      <p>There are two requirements needed prior to transferring data:</p>
+
+      <ul>
+        <li>A TACC Account</li>
+        <li>
+          Multi-Factor Authentication (MFA) pairing with the
+          <a target="_blank" href="https://portal.tacc.utexas.edu/tutorials/multifactor-authentication#tacctokenapp">TACC Token app</a>
+        </li>
+      </ul>
+
+      <p>
+        As an Portal user, you should already have access to a TACC account through the
+        <a target="_blank" href="https://portal.tacc.utexas.edu/">TACC Portal</a>
+        . If you are unsure of your TACC account username, please consult the
+        “TACC Account Request Confirmation” email you have received from TACC Accounting Staff, or a relevant contact <samp>@tacc.utexas.edu</samp>.
+      </p>
+      <p>
+        TACC now requires MFA pairing. To set up MFA, you can follow
+        <a target="_blank" href="https://portal.tacc.utexas.edu/tutorials/multifactor-authentication">this tutorial</a>.
+      </p>
+      <h4 id="command_line_tools">
+        Using Command Line Tools to Transfer and Organize Data
+      </h4>
+
+      <p>
+        A common method of transferring files between TACC resources and/or your local machine is through the command line.
+      </p>
+      <h4 id="scp_sftp_rsync">
+        <code>scp</code>, <code>sftp</code>, &amp; <code>rsync</code>
+      </h4>
+
+      <p>
+        These three command line tools are secure and can be used to accomplish data transfer. You can run these commands directly from the terminal if your local system runs Linux or macOS.
+      </p>
+      <blockquote>
+        <p>
+          Note: It is possible to use these command line tools if your local machine runs Windows, but you will need to use a ssh client (ex.
+          <a target="_blank" href="https://www.putty.org/">PuTTY</a>
+          ).
+        </p>
+        <p>
```

Comment:
To simplify the data transfer process, it is recommended that Windows users follow the <a href="#cyberduck_transfer">Transfer Data with Cyberduck</a> guide as detailed below.